Jenkinsfile

```diff
@@ -97,10 +94,12 @@ pipeline {
                                 expression { return !fileExists("src/brave/package.json") || !SKIP_INIT }
                             }
                             steps {
-                                sh """
-                                    rm -rf src/brave
-                                    npm run init -- --target_os=android
-                                """
+                                timeout(time: 1, unit: "HOURS") {
+                                    sh """
+                                        rm -rf src/brave
```

Comment:
I don't think we should be doing stuff like this. If this is necessary then there is another problem that needs to be addressed.

Reply:
is it ok to just take it out? or have to do `git clean -fxd` instead?